### PR TITLE
[bug](jsonb)fix CAST String to JSONB nullable problem

### PR DESCRIPTION
**Pull request by xiaokang**
# Proposed changes

Issue Number: close [DSIP-016: Support JSON type](https://cwiki.apache.org/confluence/display/DORIS/DSIP-016%3A+Support+JSON+type?src=contextnavpagetreemode)

## Problem summary

fix CAST String to SONB nullable problem in DEBUG mode.

## Checklist(Required)

1. Does it affect the original behavior: 
    - [ ] Yes
    - [x] No
    - [ ] I don't know
2. Has unit tests been added:
    - [x] Yes
    - [ ] No
    - [ ] No Need
3. Has document been added or modified:
    - [ ] Yes
    - [ ] No
    - [x] No Need
4. Does it need to update dependencies:
    - [ ] Yes
    - [x] No
5. Are there any changes that cannot be rolled back:
    - [ ] Yes (If Yes, please explain WHY)
    - [x] No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 35.14 seconds
 load time: 430 seconds
 storage size: 17122604992 Bytes
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20221128003341_clickbench_pr_53916.html

File: regression-test/conf/regression-conf.groovy [hunk 1]
```diff
@@ -23,11 +23,11 @@ defaultDb = "regression_test"
 // add useLocalSessionState so that the jdbc will not send
 // init cmd like: select @@session.tx_read_only
 // at each time we connect.
-jdbcUrl = "jdbc:mysql://127.0.0.1:9030/?useLocalSessionState=true"
+jdbcUrl = "jdbc:mysql://127.0.0.1:9830/?useLocalSessionState=true"
```
Comment (hello-stephen) [inline regression-test/conf/regression-conf.groovy:26]: commit by mistake?

Comment (xiaokang) [reply regression-test/conf/regression-conf.groovy:26]: yes, I will revert it

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**